instrumentation/instalambda/handler.go

```diff
@@ -157,6 +157,17 @@ func (h *wrappedHandler) triggerEventSpanOptions(payload []byte) []opentracing.S
 		}
 
 		return []opentracing.StartSpanOption{h.extractSQSTriggerTags(v)}
+	case sdkInvokeRequestType:
+		tags := opentracing.Tags{
+			"lambda.trigger": "aws:sdk.invoke",
```

Comment:
```suggestion
			"lambda.trigger": "aws:lambda.invoke",
```
Just as above, "invoke" is a very generic term and there is a chance that we'd need to _invoke_ something else :)

Reply:
I will change it